#goal
int32 number_of_cars
---
#result
int32 total_cars_cleaned
---
#feedback
float32 percent_cars_complete